src/server.mjs

```diff
@@ -664,13 +730,64 @@ async function getChunkedData(req, res, next) {
     // We're using a shared directory for this request, so leave it in place
     // when the request finishes.
     req.rmChunk = false;
-    const vgViewCall = spawn(`${VG_PATH}vg`, [
-      "view",
-      "-j",
-      `${req.chunkDir}/chunk.vg`,
-    ]);
+    // vg simplify for bed files
+    let vgSimplifyCall = null;
+    let vgViewArguments = ["view", "-j"];
+    if (req.simplify){
+      vgSimplifyCall = spawn(`${VG_PATH}vg`, ["simplify",`${req.chunkDir}/chunk.vg`,]);
+      vgViewArguments.push("-");
+      console.log("Spawning vg simplify call");
+    } else {
+      vgViewArguments.push(`${req.chunkDir}/chunk.vg`);
+    }
+     
+    let vgViewCall = spawn(`${VG_PATH}vg`, vgViewArguments);
+
     let graphAsString = "";
     req.error = Buffer.alloc(0);
+
+    // vg simplify
+    if (req.simplify){
+      vgSimplifyCall.on("error", function (err) {
+        console.log(
+          "Error executing " +
+            VG_PATH +
+            "vg " +
+            "simplify " + 
+            "- " + 
```

Comment:
This doesn't accurately report the command that was called; when we run `vg simplify` here we give it a filename as an argument and not `-`.